packages/office-ui-fabric-react/src/components/Calendar/Calendar.tsx

```diff
@@ -222,8 +222,13 @@ export class Calendar extends BaseComponent<ICalendarProps, ICalendarState> impl
 
   @autobind
   private _onGotoToday() {
-    this._navigateDay(this.props.today!);
+    let { onSelectDate, today } = this.props;
+    this._navigateDay(today!);
     this._focusOnUpdate = true;
+
+    if (onSelectDate) {
```

Comment:
Hi!  I don't see the month picker working (when it is month picker only) in the master branch either.  I can add a new task to fix this.

The selectedDate state is set to todays date, so using  this._onSelectDate(today) in the _onGotoToday() would set the selectedDate state to today again and not really do anything.